docker/puppet-agent-alpine/Dockerfile

```diff
@@ -2,17 +2,17 @@
 # build tooling
 ######################################################
 
-FROM alpine:3.9 as build
+FROM alpine:3.10 as build
```

Comment:
Bummer, looks like Travis has problems with alpine 3.10 like it had with 3.8 -- 

```
failed to solve with frontend dockerfile.v0: failed to build LLB: failed to load cache key: docker.io/library/alpine:3.10 not found
```

So this PR isn't really viable until Travis addresses that.